plugins/travis.js

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+
+// USAGE node travis.js <user> <repo> (both are case-sensitive)
+
+var request = require('request');
+
+var user = process.argv[2];
+var repo = process.argv[3];
+
+request('https://api.travis-ci.org/repos/' + user + '/' + repo + '/builds', function(error, response, body) {
+    if (error) {
+        console.log('An error occurred: ' + JSON.stringify(error));
+        process.exit(1);
```

Comment:
Perhaps if the request failed, it should make boop report something different to a failure?